### ISSUE TITLE
Create `ChromatographyConfiguration` instances and upload to Mongo

### ISSUE THREAD

**Issue by brynnz22** [microbiomedata/issues#749]
After conducting the first round of mapping, the initial draft spreadsheet for `ChromatographyConfiguration` instances have been created. What is left:
- [x] @SamuelPurvine maps the DMS slots as best as possible
- [x] @kheal draft `ChromatographyConfiguration`s in yaml form
~~- [ ] #773~~
~~- [ ] Add `ChemicalEntity` ids to `ChromatographyConfiguration`s yamls @kheal ~~
- [x] Mint `ChromatographyConfiguration` ids and add to `ChromatographyConfiguration`s yamls  @brynnz22 
- [x] Convert to json and ingest into Berkeley Mongo. @brynnz22 

LATER
- Expand modeling of `ChromatographyConfiguration` to be able to describe gradients and capture more detailed column information. _captured this on the [Berkeley-Schema Refactor Roll Out board](https://github.com/orgs/microbiomedata/projects/129_)
- Expand these using a queries:run update to add in ordered_mobile_phases information once #773 is complete

Follow up notes
After discussion - `ordered_mobile_phases.substances_used.known_as` slot that uses the `ChemicalEntity` class - is this inlined or by id? [ANSWER: by ID, see new issue in above task list]. If id, then we need to create a new issue to create these `ChemicalEntity` instances and mint ids. And then map ids back to this slot for the `ChromatographyConfiguraiton` instances.

**Comment by kheal** [comment]
Draft of `ChromatographyConfiguration` yamls is found in this draft PR
https://github.com/microbiomedata/berkeley-schema-fy24/pull/230

**Comment by brynnz22** [comment]
Blocked for now until Berkeley become production since Berkeley Mongo is not replacing production Mongo.

**Comment by brynnz22** [comment]
@kheal  Here is the last draft we landed on for the chrom configs. Not sure if you need to adjust these for the Chemical Entity modeling you and @sierra-moxon have been working on: [chrom_config_in_progress.json](https://github.com/user-attachments/files/17103077/chrom_config_in_progress.json)

**Comment by kheal** [comment]
Thanks @brynnz22.  No need to adjust yet, but it's great to have them here for reference.

**Comment by brynnz22** [comment]
closing this. It doesn't include expanding those configurations for the chemical entity stuff - not sure if you need to create another issue for that or if you have created one @kheal
